src/Stan/Inspection/AntiPattern.hs

```diff
@@ -251,3 +254,46 @@ stan0210 = mkAntiPatternInspection (Id "STAN-0210") "Slow 'for_' on ranges" (Fin
 
     forType :: PatternType
     forType = listPattern |-> ((?) |-> (?)) |-> (?)
+
+-- | 'Inspection' — slow 'length' for 'Data.Text' @STAN-0211@.
+stan0211 :: Inspection
+stan0211 = mkAntiPatternInspection (Id "STAN-0211") "'</>' for URLs" (FindAst pat)
+    & descriptionL .~ "Usage of '</>' for URLs results in the errors on Windows"
+    & solutionL .~
+        [ "Concatenate URLs with slashes '/'"
+        ]
+    & severityL .~ Error
+  where
+    pat :: PatternAst
+    pat =   opApp (httpLit ||| urlName) op (?)
+        ||| opApp (?) op urlName
+
+    op :: PatternAst
+    op = PatternAstName operator ((?) |-> filePathType |-> (?))
+      where
+        operator :: NameMeta
+        operator =  NameMeta
+            { nameMetaName       = "</>"
+            , nameMetaModuleName = "System.FilePath.Posix"
+            , nameMetaPackage    = "filepath"
+            }
+
+    filePathType :: PatternType
+    filePathType = "FilePath" `baseNameFrom` "GHC.IO" |:: []
+        ||| stringPattern
+        -- ||| primTypeMeta "[]" |:: [ charPattern ]
```

Comment:
Should we create an issue to investigate this behaviour in the future? It's really strange that it works that way... Or at least leave a node in the code as a comment?